help/destinations/catalog/email-marketing/sendgrid.md

```diff
@@ -0,0 +1,188 @@
+
+| Name | Field | Type | Description | Limits |
+|---|---|---|---|---|
+| contacts | - | Array of Objects | One or more contacts objects that you intend to add/insert. The available fields for a contact, including the required email field are described below. | Max: 30,000 items and Min: 2 items |
+| | address_line_1 | String | The first line of the address. | Max Length: 100 characters |
+| | address_line_2 | String | An optional second line for the address. | Max Length: 100 characters |
+| | alternate_emails | Array of String | Additional emails associated with the contact. | Max: 5 items and Min: 0 items |
+| | city | String | The contact's city. | MaxLength: 60 characters |
+| | country | String | The contact's country. Can be a full name or an abbreviation. | Max Length: 50 characters |
+| | email | String | The contact's primary email. This is a *mandatory* field and the value passed should be valid. | Max Length: 254 characters |
+| | first_name | String | The contact's name | Max Length: 50 characters |
+| | last_name | String | The contact's family name | Max Length: 50 characters |
+| | postal_code | String | The contact's ZIP code or other postal code. | |
+| | state_province_region | String | The contact's state, province, or region. | Max Length: 50 characters|
+| | line | String | The contact's Line account *(if any)* | |
+| | whatsapp | String | The contact's Whatsapp account *(if any)* | |
+| | facebook | String | The contact's Facebook account *(if any)* | |
+| | unique_name | String | | |
+| | phone_number | String | The contact’s phone number. | |
+
+* After completing the mappings, select **Next** to obtain a review screen indicating the audience and destination are connected.
+![](../../assets/catalog/email-marketing/sendgrid/22.png)
+
+* Select **Finish** to complete the setup.
+![](../../assets/catalog/email-marketing/sendgrid/23.jpg)
+
+## Validate data within Sendgrid {#validate}
+
+To validate that you have correctly set up the destination.
+
+1. Select **Destinations > Browse** to navigate to the list of destinations. 
+![](../../assets/catalog/email-marketing/sendgrid/25.jpg)
+
+1. Select the destination and ensure the status is **enabled**.
+![](../../assets/catalog/email-marketing/sendgrid/26.jpg)
+
+1. Switch to the **Activation data** tab, then select a **Segment Name**.
+![](../../assets/catalog/email-marketing/sendgrid/27.jpg)
+
+1. Monitor the segment summary and check the count of profiles corresponds to the count created within the dataset.
+![](../../assets/catalog/email-marketing/sendgrid/28.jpg)
+
+1. The [Sendgrid Marketing Lists API](https://api.sendgrid.com/v3/marketing/lists) is used to create unique contact lists within Sendgrid  by joining the value of the *list_name* attribute and the timestamp of the data export. Navigate to the Sendgrid site and check if the new contact list conforming to the name pattern is created.
```

Comment:
Please provide a link to the documentation page for the API. The link you provided shows this for me: 
![Screen Shot 2022-04-01 at 5 02 55 PM](https://user-images.githubusercontent.com/19704189/161340886-79101930-ffb0-41ec-9dfa-b8ce69298ed4.png)